Oqtane.Client/Program.cs

```diff
@@ -164,6 +110,45 @@ private static async Task LoadClientAssemblies(HttpClient http)
             }
         }
 
+        private static void RegisterModuleServices(Assembly assembly, IServiceCollection services)
+        {
+            var implementationTypes = assembly.GetInterfaces<IService>();
+            foreach (var implementationType in implementationTypes)
+            {
+                if (implementationType.AssemblyQualifiedName != null)
+                {
+                    var serviceType = Type.GetType(implementationType.AssemblyQualifiedName.Replace(implementationType.Name, $"I{implementationType.Name}"));
+                    services.AddScoped(serviceType ?? implementationType, implementationType);
+                }
+            }
+        }
+
+        private static void RegisterClientStartups(Assembly assembly, IServiceCollection services)
+        {
+            var startUps = assembly.GetInstances<IClientStartup>();
+            foreach (var startup in startUps)
+            {
+                startup.ConfigureServices(services);
+            }
+        }
+
+        private static async Task SetCultureFromLocalizationCookie(IServiceProvider serviceProvider)
+        {
+            var jsRuntime = serviceProvider.GetRequiredService<IJSRuntime>();
+            var interop = new Interop(jsRuntime);
+            var localizationCookie = await interop.GetCookie(CookieRequestCultureProvider.DefaultCookieName);
+            var culture = CookieRequestCultureProvider.ParseCookieValue(localizationCookie).UICultures[0].Value;
```

Comment:
today if I try to initialize Oqtane (first start) and use "Runtime": "WebAssembly" I'll receive something like
![image](https://user-images.githubusercontent.com/4517566/121971837-5d518c00-cd82-11eb-8552-636e6bddf01d.png)

this is the fix
`var culture = CookieRequestCultureProvider.ParseCookieValue(localizationCookie)?.UICultures?[0].Value;`

would not like to add separate PR in order to avoid merge conflicts
@hishamco could you please modify your PR

Reply:
Openning a new issue is better for tracking

Reply:
agree, done